src/globals/scss/helpers/_arrow.scss

```diff
@@ -19,10 +19,10 @@
 //
 // We need both because older browsers do not support clip-path.
 //
-// @param {Direction} $direction - Direction for arrow: up, right, down, left.
+// @param {Direction} $direction - Direction for govuk-shape-arrow: up, right, down, left.
 // @param {Base} $base - Length of the triangle 'base' side
 // @param {Height} $height - Height of triangle. Omit for equilateral.
-@mixin arrow($direction, $base, $height: null) {
+@mixin govuk-shape-arrow($direction, $base, $height: null) {
```

Comment:
If we're going to change the name of this to shape-arrow then can we update the filename to `_shape-arrow.scss` too please?